services-directions-models/src/main/java/com/mapbox/api/directions/v5/utils/FormatUtils.java

```diff
@@ -240,4 +241,36 @@ public static String formatPointsList(@Nullable List<Point> points) {
     }
     return join(";", coordinatesToJoin);
   }
+
+  /**
+   * Converts String list with annotation values to a string ready for API consumption.
+   * Possible values are:
+   * {@link DirectionsCriteria#ANNOTATION_DURATION}
+   * {@link DirectionsCriteria#ANNOTATION_DISTANCE}
+   * {@link DirectionsCriteria#ANNOTATION_SPEED}
+   * {@link DirectionsCriteria#ANNOTATION_CONGESTION}
+   * {@link DirectionsCriteria#ANNOTATION_MAXSPEED}
+   *
+   * @param annotations a list representing annotation to each coordinate.
+   * @return a formatted string.
+   */
+  @Nullable
+  public static String formatAnnotations(@Nullable List<String> annotations) {
+    if (annotations == null || annotations.isEmpty()) {
+      return null;
+    }
+
+    for (String annotation : annotations) {
+      if (annotation != null
+          && !annotation.equals(DirectionsCriteria.ANNOTATION_DURATION)
```

Comment:
What happens if in the future we want to add a new annotation in `DirectionsCriteria`? I believe we shouldn't check if the annotations passed in are not equal to the current ones available because if a new annotation is added we'll need to change this code. How can we avoid that situation? What do you think?

Reply:
we can't iterate on them, they are just static constants.
I can remove that check, docs have full description according valid params.